sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -378,6 +378,12 @@ def analyze_sentiment(  # type: ignore
         :type documents:
             list[str] or list[~azure.ai.textanalytics.TextDocumentInput] or
             list[dict[str, str]]
+        :keyword bool show_aspects: Whether to conduct more granular analysis around the aspects of
```

Comment:
We should add a [.. versionadded](https://www.sphinx-doc.org/en/1.7/markup/para.html#directive-versionadded) (or possibly a [.. note](https://www.sphinx-doc.org/en/1.7/markup/para.html#directive-note) directive that indicates in which API version(s) this parameter is available. We should see what the generated docs look like for each since our docs pipelines are customized and I don't think we've used either up until now...

Reply:
added this code to the PR. While this is not generating great right now (@johanste you mentioned the indentation is off etc), @scbedd is looking into better rendering (thanks Scott!!!!)